R/sanitize_code.R

```diff
@@ -80,13 +80,12 @@ fill_block_comments <- function(lns, fill_with = " ") { # nolint: object_usage_l
   n_close <- sum(valid_syms[["type"]] == "close")
   # Fails if number of `/*` and `*/` are different.
   if (n_open != n_close) {
-    ui_throw(
+    cli::cli_abort(c(
       "Malformed comments.",
-      c(
-        bullet_x("Number of start {.code /*} and end {.code */} \\
-               delimiters are not equal."),
-        bullet_i("Found {n_open} occurence{?s} of {.code /*}."),
-        bullet_i("Found {n_close} occurence{?s} of {.code */}.")
+       "x" = "Number of start {.code /*} and end {.code */} \\
```

Comment:
```suggestion
      "x" = "Number of start {.code /*} and end {.code */} \\
```

Reply:
Ran everything through style! Should all be spick and span!